backend/functions/lib/adapters/utils/sessionCache.ts

```diff
@@ -0,0 +1,6 @@
+import { CamsSession } from '../../../../../common/src/cams/session';
+import { ApplicationContext } from '../types/basic';
+
+export interface SessionCache {
```

Comment:
Is this file path right for this interface definition??

Reply:
Maybe. This placement and the naming of things were sort of just done as part of the TDD process, but we didn't ever get together and discuss whether they should move/change.

Reply:
placement is correct, it is closest to the file that uses it